package.json

```diff
@@ -12,6 +12,7 @@
     "generate:assets": "echo \"No assets to configure\"",
     "generate:meetings": "node scripts/build-meetings.js",
     "generate:videos": "node scripts/build-newsroom-videos.js",
+    "generate:tools": "node scripts/build-tools.js && node scripts/tools/combine-tools.js",
```

Comment:
imho `combine-tools` should be called from `build-tools` as it is part of the tools build process

Reply:
I tried this before, but if you try to import `combine-tools` in `build-tools` file, and then call it, it will result in lot of import errors. I would better suggest you to look into the functionality and then you will get why I actually made the script in this way.

Reply:
`combine-tools` is basically about combining manual and automated too.

I assume that when trying to call it from `build-tools` you get error because of `const automatedTools = require("../../config/tools-automated.json")`. And it makes sense because the file either do not exist, or is simply not updated on a runtime. 

`build-tools`  is where the `tools-automated.json` is created. So in `combine-tools` do not `require` `tools-automated.json` but just enable main function to accept it as an argument, and pass `tools-automated.json` object to `combine-tools` when you call if from `build-tools`